modules/core/src/lib/attribute.js

```diff
@@ -129,6 +130,26 @@ export default class LayerAttribute extends Attribute {
     return updated;
   }
 
+  setGenericValue({props}) {
+    const state = this.userData;
+    let value = props[state.accessor];
+
+    if (value === undefined || typeof value === 'function') {
```

Comment:
It seems a little unusual to do this check inside of this function rather than while iterating over the attributes.

What case is the `undefined` value covering?

Reply:
`state.accessor` could be undefined or an array of strings.